### PR TITLE
Add signup page.

### PR DESCRIPTION
**Pull request by manojkumar8400**
i had added signup page, also added the facebook, twitter, google sign feature to it.

### CODE REVIEW

**Review by KaustubhM1997 — APPROVED** [review]
The styles for the icons and button in the navbar haven't been rendered correctly. Also, the signup card is overlapping with the navbar a bit which you might want to change. The rest looks good!

File: signup-layout/signup.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+.login-link {
+    text-align: center;
+}
+
+.or-option {
+    text-align: center;
+}
+
+.signup-way {
+    text-align: center;
```
Comment (siddhigate) [inline signup-layout/signup.css:54]: As you've used `text-align: center` at multiple places, you can create a utility class called `.txt-center` and use it wherever required. Just a suggestion

**Review by siddhigate — APPROVED** [review]

File: signup-layout/signup.css (hunk 1 above)
Comment (manojkumar8400) [reply signup-layout/signup.css:54]: ok, I will work on it.

**Comment by manojkumar8400** [comment]
> The styles for the icons and button in the navbar haven't been rendered correctly. Also, the signup card is overlapping with the navbar a bit which you might want to change. The rest looks good!

Ok, I will work on it.

File: signup-layout/signup.css (hunk 1 above)
Comment (manojkumar8400) [reply signup-layout/signup.css:54]: ok, I will change it.

**Pull request merged**